features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
```

Comment:
Your suggestion is reasonable but cannot be executed as a single line and needs to be broken up into three lines. The reason for this is that the split call does not allocate nil storage for the collection elements so their values cannot be interrogated and reset to an empty string.  I verified this in a debug session and saw where a page value of  " 	IT  / "  had chart=IT and org=nil after the single line code was executed but had the values of chart=IT and org='' after the following was executed:
    chart,org = req_page.chart_org_readonly.split('/')
    chart = chart.nil? ? '' : chart.strip
    org = org.nil? ? '' : org.strip